numba_dpex/tests/dpjit_tests/dpnp/test_dpnp_empty.py

```diff
@@ -67,3 +91,19 @@ def func(shape):
     dummy_tensor = dpctl.tensor.empty(shape)
```

Comment:
We do not need to create another array to check if the `dtype`s are the same, we can just check if the returned array comes with the `dtype` that we provided in the argument.